vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -4,20 +4,20 @@
       {
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "versions": [
-          "1.19.11-hotfix.20210526.3",
           "1.19.11-hotfix.20210823",
+          "1.19.11-hotfix.20210823.1",
```

Comment:
What's the principle of keeping and removing an old version?

Reply:
this PR is special.
all due to we have no vhd release this week.
so we jump the image tag.